### PR TITLE
update nofrac case file

### PR DESCRIPTION
**Pull request by jkuang-exploro**
inline post update for config.post
update nofrac case file

### CODE REVIEW

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA @jiandewang This is a draft PR and probably needs a few iterations of modification. Please review.

**Comment by jiandewang** [comment]
@JianKuang-NOAA  
I just tried on HERA, waveinit failed for both frac=T or F cases
see log file on orion at
/work/noaa/marine/Jiande.Wang/For-others/For-Jian

**Comment by jiandewang** [comment]
@JianKuang-NOAA I will try on ORION for future work (just realize you don't have access to hera)

File: workflow/cases/coupled_free_forecast_nofrac_wave.yaml [hunk 1]
```diff
@@ -35,6 +36,7 @@ case:
     DELTIM: 450
     SEEDLET: 10
     CCPP_SUITE: FV3_GFS_v15p2_coupled
+    CPL_ATMIC: CFSRfracL64
```
Comment (JessicaMeixner-NOAA) [inline workflow/cases/coupled_free_forecast_nofrac_wave.yaml]: For non-fractional grid this should be just CFSR not CFSRfracL64

File: workflow/cases/coupled_free_forecast_wave.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ case:
     print_esmf: .true.
     nems_temp: 'cpld_wave'
     mom6ic_prepared: .false.
-    inline_post: .false. 
+    inline_post: .true.
```
Comment (JessicaMeixner-NOAA) [inline workflow/cases/coupled_free_forecast_wave.yaml:19]: inline post should be false for all cases right now unless inline post updates have been made?

**Comment by JessicaMeixner-NOAA** [comment]
@JianKuang-NOAA, I'm back from AL and just made a few comments on the code changes themselves.  Are you ready for me and/or @jiandewang to make runs as well?

File: workflow/cases/coupled_free_forecast_wave.yaml (hunk 2 above)
Comment (jkuang-exploro) [reply workflow/cases/coupled_free_forecast_wave.yaml:19]: This PR also includes updates to inline post functionalities. Let's also test that part.

**Comment by jiandewang** [comment]
@JianKuang-NOAA let me and Jessica know when you believe it's mature for us to make test run

**Comment by jkuang-exploro** [comment]
@jiandewang @JessicaMeixner-NOAA I think the error that Jiande has got, is because the fix file folder has not been updated on Hera, respectively. 

@JessicaMeixner-NOAA Would you please help me update the fix file folders for Hera platform in addition to Orion?

**Comment by JessicaMeixner-NOAA** [comment]
So everything works for you @JianKuang-NOAA  on orion now?  If so, I'll start testing on orion first, and then will look at hera issues. The fix files should be the same on both platforms so I'm not really sure what you mean by " the fix file folders for Hera platform in addition to Orion?"

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA I remembered you told me that additional link folder was included, in a recent PR and the link script was updated. However, Jiande's error indicates that one WW3 *.inp file was not invoked properly during waveinit job. So I doubt if the actual contents of the new fix file are indeed there, on Hera. So worth to check it up.

**Comment by JessicaMeixner-NOAA** [comment]
@JianKuang-NOAA I'm starting testing on orion first, I need to test the following cases: 
coupled_free_forecast_nofrac_wave.yaml which should be running again (it was broken before) and
coupled_free_forecast_wave.yaml  which should now be running with inline post, correct?

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA Yes I think so.

**Comment by jiandewang** [comment]
@JianKuang-NOAA both fractional and non-fractional cases finished OK on HERA.
One question: for fractional run it is with inlinepost on, but I still saw all those post log files, is this suppose to be ? In other words, what kind of different behaviors shall we expect when job running ?
I will check results on Tuesday.

**Comment by JessicaMeixner-NOAA** [comment]
I ran tests (no wave, non-fractional, frac grid w/inline post, frac grid no inline post) on orion and here are a few comments: 

* All tests ran to completion 
* the archive job succeeded, but should have failed  
* There is a weird folder $COMROOT/gfs.$YYYYMMDD/00/atmos/00 that links to a non-existent "place_holder" folder that it seems to me should not be there.  I couldn't find where this was getting created though.  Might not be from this PR
* I'm still looking into the differences between the inline and no-inline post directories to determine if the differences are as expected or not

**Comment by jiandewang** [comment]
one strange thing I found is after post all pgb files are stored at /EXPROOT/noscrub/ID/archive/ instead of COMROOT/.../.../atmos
here ID is your login ID
I checked several output files, they look fine

**Comment by jiandewang** [comment]
for fractional-on cases, what I saw is after fcst job finished, it generated files like "gfs.t00z.sfluxgrbf006.grib2". Then it will still launch all those post jobs. Is it suppose to be like this way (I don't know what exactly inline post is).

**Comment by JessicaMeixner-NOAA** [comment]
@jiandewang yes the separate post is still supposed to run even with inline post

**Comment by junwang-noaa** [comment]
@jiandewant please make sure you are not regenerating the master files in
post job, post will only need to generate low resolution post files

**Comment by jkuang-exploro** [comment]
@jiandewang I noticed the"/EXPROOT/noscrub/ID/archive" issue. This is wrong. I am working on a fix

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA The softlink to nowhere that mentioned by @jiandewang is probably in the waveprep job since I saw here in the log of waveprep job (This happens every time so you could see it in any experiment):

/bin/ln -sf /work/noaa/marine/place_holder/gfs.20130401/00/ /work/noaa/marine/Jian.Kuang/comroot/pr222/gfs.20130401/00/atmos/

**Comment by JessicaMeixner-NOAA** [comment]
@JianKuang-NOAA thanks for pointing that out! Can you point me to the wave prep log you saw this in so I can back trace where it is in the scripts and remove it.  Thanks for finding this!

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA Please take a look at this log:

/work/noaa/marine/Jian.Kuang/comroot/pr222/logs/2013040100/gfs.prep.waveprep.log

**Comment by JessicaMeixner-NOAA** [comment]
Looking in the wave prep log all of the variables below have a placeholder definition: 

DMPDIR=/work/noaa/marine/place_holder
RTMFIX=/work/noaa/marine/place_holder
BASE_GIT=/work/noaa/marine/place_holder
BASE_SVN=/work/noaa/marine/place_holder ---- do we need this variable anymore? 
HOMEobsproc_prep=/work/noaa/marine/place_holder/obsproc/obsproc_global.v3.2.3
HOMEobsproc_network=/work/noaa/marine/place_holder/obsproc/obsproc_global.v3.2.3
BASE_VERIF=/work/noaa/marine/place_holder/verif/global/tags/vsdb

but what is creating that folder is: 
    $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}

so the DMPDIR directory definition is what is triggering this.   I'm going to create a separate issue to track and fix this, it was not introduced by this PR and so doesn't need to be fixed in this PR either.

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA OK. I agree. Does the inline post work as desired?

**Comment by jiandewang** [comment]
what I noticed is fcst job will generate "sflux*grib2" files, then all post job will be launched after this (Jessica believed this is what it suppose to be). My test on HERA shows the workflow works end-to-end and results look fine though there are some minor issues such as strange directory and archiving.
I agree with @JessicaMeixner-NOAA on trying to address other issues in a separate PR.

**Comment by JessicaMeixner-NOAA** [comment]
Is this issue going to be fixed in another PR or in this PR @JianKuang-NOAA ?

> @jiandewang I noticed the"/EXPROOT/noscrub/ID/archive" issue. This is wrong. I am working on a fix

**Comment by jkuang-exploro** [comment]
@JessicaMeixner-NOAA Sorry I forgot to push this fix after worked it out. Now pushed. Please check.

**Comment by JessicaMeixner-NOAA** [comment]
I'll run one more test w/this latest update.  Thanks @JianKuang-NOAA

**Comment by jiandewang** [comment]
@JessicaMeixner-NOAA let me know your run dir so that we can double check that, I am not going to repeat on HERA

**Comment by JessicaMeixner-NOAA** [comment]
@jiandewang do you want my run or COM dir?   The com will be: /work/noaa/marine/jmeixner/p6pr222/noinline04/COMROOT/waves05 for the case file coupled_free_forecast_wave.yaml

The forecast job just started running.

**Comment by jiandewang** [comment]
@JessicaMeixner-NOAA COM dir is enough for me to check, thanks

**Comment by jiandewang** [comment]
I saw the noscrub dir is still there, but that's from Jessica's previous run, not this run
looking at COM dir, all output exist

**Comment by JessicaMeixner-NOAA** [comment]
Yeah everything looks fine except for I don't know why the archive job is completing on orion -- that should fail as we have no HPSS.  It used to fail.

**Review by JessicaMeixner-NOAA — APPROVED** [review]
Thanks @JianKuang-NOAA  

we'll address any archive job issues later as well as the place_holder linking.

**Pull request merged**